### ISSUE TITLE
wrong and missing PGN 129029 data

### ISSUE THREAD

**Issue by Tigrecito** [SignalK/signalk-to-nmea2000#80]
Hi,

navigation.gnss.antennaAltitude
navigation.gnss.horizontalDilution
navigation.gnss.satellites

are visible in data browser with proper values but:

Number of SVs is always 16 and HDOP is always 0.64
Also Altitude is missing in totaly.

This is the signalk-to-nmea2000 log:
{"pgn":129029,"Date":19607,"Time":41594,"Latitude":50.86987516666667,"Longitude":15.29382105,"GNSS type":"GPS+SBAS/WAAS","Method":"DGNSS fix","Integrity":"No integrity checking","Number of SVs":16,"HDOP":0.64,"Geoidal Separation":-0.01,"Reference Stations":1,"Reference Station Type":"GPS+SBAS/WAAS","Reference Station ID":7}

**Comment by sbender9** [comment]
I think you have the wrong repository. Should be n2k-signalk.

**Comment by Tigrecito** [comment]
I don´t think so .. I want to push GPS data from signal k server into NMEA 2000
using the Signal K to NMEA 2000 plugin.

**Comment by sbender9** [comment]
Yep. Sorry, misinterpreted the issue at first...

**Comment by aviatorhh** [comment]
FYI, here the problem shown with GPSD forming NMEA0183 sentences out of the can messages (NMEA2000).
![ss](https://github.com/SignalK/signalk-to-nmea2000/assets/13214417/70bb7178-e3e2-4c0c-8d0a-d8aa63d09055)
No sat view data. Position data remove by me.

**Comment by jacques-cd** [comment]
A started looking at this issue because none of the nav data was displaying on my Garmin instrument displays, so started making some fixes that improve this in #98 - this is still a bit of a WIP, but happy to take feedback on the PR.
